sentry_sdk/integrations/aws_lambda.py

```diff
@@ -25,6 +27,49 @@
 
     F = TypeVar("F", bound=Callable[..., Any])
 
+# Constants
+TIMEOUT_THRESHOLD_MILLIS = 1500  # Minimum time required to capture TimeoutError
+SECONDS_CONVERSION_FACTOR = 1000.0
+
+
+def _wrap_init_error(init_error):
+    # type: (F) -> F
+    def sentry_init_error(*args, **kwargs):
+        # type: (*Any, **Any) -> Any
+
+        # Fetch Initialization error details from arguments
+        error = json.loads(args[1])
+
+        hub = Hub.current
+        integration = hub.get_integration(AwsLambdaIntegration)
+        if integration is None:
+            return init_error(*args, **kwargs)
+
+        # If an integration is there, a client has to be there.
+        client = hub.client  # type: Any
+
+        with hub.push_scope() as scope:
+            with capture_internal_exceptions():
+                scope.clear_breadcrumbs()
+            try:
+                # Checking if there is any error/exception which is raised in the runtime
+                # environment from arguments and, re-raising it to capture it as an event.
+                if error.get("errorType"):
+                    exc_info = sys.exc_info()
+                    reraise(*exc_info)
```

Comment:
Why reraise to catch again instead of calling `sys.exc_info` once? Does this change the stacktrace somehow?

Reply:
Nope, this doesn't change the stacktrace.  
I got your point. I'll check once and capture the details without reraising.